src/main.js

```diff
@@ -0,0 +1,31 @@
+import { createProfileRatingTemplate } from './view/profile-rating-view.js';
+import { createSiteMenuTemplate } from './view/site-menu-view.js';
+import { createSortMenuTemplate } from './view/sort-menu-view.js';
+import { createFilmsTemplate } from './view/films-view.js';
+import { createFilmCardTemplate } from './view/film-card-view.js';
+import { createShowMoreButtonTemplate } from './view/show-more-button-view.js';
+import { createFilmsCountTemplate } from './view/films-count-view.js';
+import { createPopupTemplate } from './view/popup-view.js';
+import { renderTemplate, RenderPosition } from './render.js';
+
+const TASK_COUNT = 5;
+const siteHeaderElement = document.querySelector('.header');
```

Comment:
```suggestion
const TASK_COUNT = 5;

const siteHeaderElement = document.querySelector('.header');
```

объявление переменных и констант это два разных логических блока